buildtest/buildsystem/base.py

```diff
@@ -455,31 +436,35 @@ def run_tests(self, testfile):
             self.metadata.get("rundir"), "%s.out" % self.build_id
         )
 
-        # Run the test file, print output to file
-        with open(run_output_file, "w") as fd:
+        # Keep an error file
+        run_error_file = os.path.join(
+            self.metadata.get("rundir"), "%s.err" % self.build_id
+        )
 
-            fd.write("Test Name:" + self.build_id + "\n")
-            fd.write("Return Code: %s \n" % command.returncode)
+        # write output of test to .out file
+        with open(run_output_file, "w") as fd:
+            fd.write("\n".join(out))
 
-            if out:
-                fd.write("---------- START OF TEST OUTPUT ---------------- \n")
-                fd.write("\n".join(out))
-                fd.write("------------ END OF TEST OUTPUT ---------------- \n")
-            if err:
-                fd.write("---------- START OF TEST ERROR ---------------- \n")
-                fd.write("\n".join(err))
-                fd.write("------------ END OF TEST ERROR ---------------- \n")
+        # write error from test to .err file
+        with open(run_error_file, "w") as fd:
+            fd.write("\n".join(err))
 
         result["RETURN_CODE"] = command.returncode
         result["END_TIME"] = self.get_formatted_time("end_time")
 
         # Print the test result for the user
         if command.returncode == 0:
-            print("{:<40} {}".format("[RUNNING TEST]", "PASSED"))
+            print(
+                "{:<30} {:<30} {:<30} {:<30}".format(
+                    self.config_name, self.name, "PASSED", self.config_file
+                )
+            )
         else:
-            print("{:<40} {}".format("[RUNNING TEST]", "FAILED"))
-
-        print("Writing results to " + run_output_file)
+            print(
```

Comment:
Should this be FAILED if the return code isn't 0?

Reply:
good catch that was typo on my end due to copy-paste.